docs-v2/integrations/all/brex-api-key.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Brex API key
+sidebarTitle: Brex API key
+---
+
+API configuration: [`Brex API key`](https://nango.dev/providers.yaml)
```

Comment:
Let's not create another docs page here, but rather re-use the existing [Brex](https://docs.nango.dev/integrations/all/brex) one. 

Here's an example of having multiple auth modes on a single docs page for [Gong](https://docs.nango.dev/integrations/all/gong).

Reply:
Hi Bastien, thank you for your insights. I have just updated my pull request. If there are any further changes or feedback you'd like me to address, please let me know.